packages/maskbook/package.json

```diff
@@ -32,7 +32,7 @@
     "@openzeppelin/contracts": "^3.2.0",
     "@popperjs/core": "*",
     "@servie/events": "^3.0.0",
-    "@snapshot-labs/snapshot.js": "^0.1.8",
+    "@snapshot-labs/snapshot.js": "github:snapshot-labs/snapshot.js#master",
```

Comment:
Does this package need a build step? If so, please build it and pack the result as a tgz file and install from tgz.

Reply:
No, [snapshot website](https://github.com/snapshot-labs/snapshot/blob/develop/package.json#L30) use the same 'github:snapshot-labs/snapshot.js#master'

Reply:
how do u think @guanbinrui

Reply:
Why not use the newest ver? [v0.1.9](https://registry.npmjs.org/@snapshot-labs/snapshot.js/-/snapshot.js-0.1.9.tgz).

Reply:
It's easier to debug that I can read their code when keep the same with [snapshot website](https://github.com/snapshot-labs/snapshot/blob/develop/package.json#L30)

Reply:
Better not to use pkg from gh directly. It's easy to break.

Reply:
Actually it's version locked in the lock file by commit hash

Reply:
@Jack-Works I see. There is no problem with me.